Sources/Core/TextEditor/ViewModel/TextEditorViewModel.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  TextEditorViewModel.swift
+//  SparkEditor
+//
+//  Created by robin.lemaire on 05/09/2024.
+//  Copyright © 2024 Adevinta. All rights reserved.
+//
+
+import Foundation
+import SparkTheming
+
+// sourcery: AutoPublisherTest, AutoViewModelStub
+final class TextEditorViewModel: TextInputViewModel {
+
+    // MARK: - Published Properties
+
+    @Published private(set) var isPlaceholder: Bool = false
+    @Published private(set) var shouldUpdateVerticalSpacing: Int = 0
```

Comment:
Looks like it should be a Boolean

Reply:
It is not a boolean but you're right, the naming is not good. I renamed to updateVerticalSpacingCounter. It is Ok for you ?

Reply:
yeah sure :)